converters/mesh2vol.cpp

```diff
@@ -137,14 +146,18 @@ int main( int argc, char** argv )
    unsigned int margin  {0};
    unsigned int separation {6};
    unsigned int resolution {128};
- 
+   unsigned char fillValue {128};
+   bool unitScale {false};
+   
    app.description("Convert a mesh file into a 26-separated or 6-separated volumetric voxelization in a given resolution grid. \n Example:\n mesh2vol ${DGtal}/examples/samples/tref.off output.vol --separation 26 --resolution 256 ");
    
    app.add_option("-i,--input,1", inputFileName, "mesh file (.off)." )
      ->required()
      ->check(CLI::ExistingFile);
    app.add_option("-o,--output,2", outputFileName, "filename of ouput volumetric file (vol, pgm3d, ...).",true);
    app.add_option("-m,--margin", margin, "add volume margin around the mesh bounding box.");
+   app.add_flag("-d,--objectDomainBB", unitScale, "use the digitization space defined from bounding box of input mesh. If seleted, the option --resolution will have no effect.");
+   app.add_option("-f,--fillValue", fillValue, "change the default value of filling value of the volumetric image.");
```

Comment:
```suggestion
   app.add_option("-f,--fillValue", fillValue, "change the default output  volumetric image value in [1...255].");
```
maybe add a check for the fill value ?

Reply:
thanks adding to batch, yes I add the check too